docker-compose.dev.yml

```diff
@@ -0,0 +1,37 @@
+version: '3'
+
+services:
+
+  db:
+    image: postgres:9.4
+    volumes:
+      - ./config/application_user.sql:/docker-entrypoint-initdb.d/application_user.sql
+    env_file:
+      - dev.env
+
+  web:
+    build:
+      context: ./
+      dockerfile: Dockerfile.dev
+    working_dir: /srv/profiles
+    volumes:
+      - .:/srv/profiles
```

Comment:
the custom `app.cfg` is not committed (it's in `.gitignore`) but it gets into the container from here, so unless it's very specific it's time to start committing with a name like `app.cfg.dev` or similar?

Reply:
I have added this change.